### PR TITLE
Feature/LocateMeButton

### PR DESCRIPTION
**Pull request by JanRaadts**

### CODE REVIEW

File: components/Hooks/useGeoLocation.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      });
+    }
+
+    navigator.geolocation.getCurrentPosition(onSuccess, onError);
+  }, []);
+
+  return location;
+};
+
+export default useGeoLocation;
```
Comment (julialoeschel) [inline components/Hooks/useGeoLocation.js:37]: nice!

File: components/Map.js [hunk 2]
```diff
@@ -51,6 +65,14 @@ export default function Map() {
           );
         })}
       </StyledMapContainer>
+      <StyledPositionBtn onClick={handleSetView}>
+        <Image
+          src={locateMeButton}
+          alt="locateMeButton"
```
Comment (julialoeschel) [inline components/Map.js:71]: try to give real hints, what the button does. Dont use Buttum in the alt text. 
"locate me" or "locate my position" would work here.

File: components/Map.js [hunk 3]
```diff
@@ -51,6 +65,14 @@ export default function Map() {
           );
         })}
       </StyledMapContainer>
+      <StyledPositionBtn onClick={handleSetView}>
```
Comment (julialoeschel) [inline components/Map.js:68]: in your prev this doesnt work for me :/

**Review by julialoeschel — COMMENTED** [review]
good job 🐜 
the locate  me thing doesnt work fo me, but Im not sure if it is me or your code ;) 
the code looks ok to me.

really good work there!

File: components/Map.js (hunk 2 above)
Comment (JanRaadts) [reply components/Map.js]: i changed it :)

File: components/Map.js (hunk 3 above)
Comment (JanRaadts) [reply components/Map.js]: I add an if else block, if the app does not be able to use your location, there will be an altert on your browser :)

**Pull request merged**